Cargo.toml

```diff
@@ -27,21 +31,20 @@ wayland = [
 
 # passing on all the image features. AVIF is not supported because it does not
 # support decoding, and that's all we use `Image` for.
-bmp = []
-dds = []
-dxt = []
-farbfeld = []
-gif = []
-jpeg = []
-png = []
-ico = []
-tiff = []
-webp = []
-tga = []
-hdr = []
-image_png = []
-jpeg_rayon = []
-pnm = []
+bmp = ["image/bmp", "image"]
```

Comment:
I'm struggling to locate where the image crate is used (apart from as a re-export)

Admittedly, this is on mobile, so my code searching isn't quite up to scratch